app/controllers/api/v2/allocations_controller.rb

```diff
@@ -42,6 +44,15 @@ def update
         end
       end
 
+      def destroy
```

Comment:
'delete' or 'destroy'? Your PR title says 'delete'

Reply:
https://stackoverflow.com/questions/22757450/difference-between-destroy-and-delete